mne/epochs.py

```diff
@@ -3106,3 +3106,32 @@ def average_movements(epochs, head_pos=None, orig_sfreq=None, picks=None,
     _remove_meg_projs(evoked)  # remove MEG projectors, they won't apply now
     logger.info('Created Evoked dataset from %s epochs' % (count,))
     return (evoked, mapping) if return_mapping else evoked
+
+
+def segment_raw(raw, segment_length=1., verbose=True):
```

Comment:
I don't have a strong opinion here, I quite often use segmentation but maybe it is not common enough to expose it. I can add the `_` prefix if you prefer.